django/core/checks/templates.py

```diff
@@ -21,3 +26,13 @@ def check_setting_app_dirs_loaders(app_configs, **kwargs):
         if 'loaders' in conf.get('OPTIONS', {}):
             passed_check = False
     return [] if passed_check else [E001]
+
+
+@register(Tags.templates)
+def check_string_if_invalid_is_string(app_configs, **kwargs):
+    passed_check = True
+    for conf in settings.TEMPLATES:
+        string_if_invalid = conf.get('OPTIONS', {}).get('string_if_invalid', "")
+        if not isinstance(string_if_invalid, str):
```

Comment:
`from django.utils import six`
(by the way, try to do a few checks locally and not push a lot of changes to GitHub in quick succession since every push triggers a Jenkins build -- I canceled a few extra builds from this PR a couple hours ago)